### PR TITLE
Fix 875 "FreeOrion version string should be more accessible"

### LINKED ISSUE #875: FreeOrion version string should be more accessible
### Environment

- **FreeOrion Version**: master
- **Operating System**: any
- **Graphic card used**: -
- **Fetched as**
  - Compiled from source
### Description

Split of from #872.

When reporting issues on github, users are asked to give the version number of the version they experience the bug/issue.  However the user is forced to manually copy the version string because there is no way to select or output the version string easily.
### Expected Result

The user should be able to select and <Ctrl+C> the version string into the clipboard. Also when right clicking the version string there should be a context menu containing a `copy to clipboard` entry.
Additionally the user should be able to execute `./freeorion --version` on command line. This should print out the version string on the standard output and terminate the application immediately with exit=0.
#### Steps to reproduce

- Start freeorion.
- Select or right click version string
- Nothing happens

Alternative
- Start `freeorion` on command line with `freeorion --version`
- `freeorion` does start regularly without printing a version string

### PR DESCRIPTION
**Pull request by adrianbroher**
This pull request implements various features required to fix #875.

A review would be appreciated.

Also thanks @geoffthemedio for implementing the 'copy' context menu just a few days ago.  All the heavy lifting was already done.

### CODE REVIEW

**Comment by geoffthemedio** [comment]
A few days ago?

**Comment by adrianbroher** [comment]
You committed the change on the 2016-07-17, didn't you?

**Comment by Vezzra** [comment]
Well, "few" does not mean the same to everyone, does it? 😉

**Comment by geoffthemedio** [comment]
apparently not; a few days would generally be 3-5 days for me.

**Comment by adrianbroher** [comment]
Rebased to resolve merge conflict with #878.

**Comment by geoffthemedio** [comment]
Could just output the version string to stdout every time any of the freeorion binaries start...

File: client/human/chmain.mm [hunk 1]
```diff
@@ -29,5 +29,11 @@ int main(int argc, char *argv[])
         return 0;   // quit without actually starting game
     }
 
+    // did the player request the version output?
+    if (GetOptionsDB().Get<bool>("version")) {
+        std::cout << "FreeOrionCH " << FreeOrionVersionString() << std::endl;
```
Comment (Vezzra) [inline client/human/chmain.mm:35]: You forgot to include `util/Version.h` here in `chmain.mm`, resulting of course in a compile error on OSX. I [created a PR](https://github.com/adrianbroher/freeorion/pull/3) for your branch with the fix.

**Comment by Vezzra** [comment]
Aside from the missing include (see above) works as intended. Code looks fine too, so as far as I'm concerned, go ahead and merge.

**Comment by adrianbroher** [comment]
> I created a PR for your branch with the fix.

Thanks, reworded and applied to PR.

**Comment by Vezzra** [comment]
Anything preventing this from being merged? Looks, builds and runs fine for me, so, @adrianbroher, I think you can go ahead and merge.

Assuming that you're going to do that soon and because I don't want to override  your decision to tag this for the release, I'm going to leave the milestone as is (for the issue the PR addresses too), although it's not exactly something critical.

**Pull request merged**